### ISSUE TITLE
WPILib libraries not available in Python

### ISSUE THREAD

**Issue by StealthySemicolon** [wpilibsuite/FRCVision-pi-gen#153]
I am trying to put number sliders on ShuffleBoard to tune HSV filter values, and I am doing all of this in Python. However, I get a module not found error while trying to import wpilib.shuffleboard.

**Comment by PeterJohnson** [comment]
That’s correct.  The only Python libraries currently supported are for NetworkTables and CameraServer.  With the RobotPy changes for 2020 it may be possible to bring in more.

**Comment by PeterJohnson** [comment]
Note you can add stuff to shuffleboard in your robot program instead and then read those NetworkTable keys directly in Python.

**Comment by virtuald** [comment]
RobotPy's shuffleboard stuff isn't working for 2020 yet, but hoping to tackle that this week.
